CMakeLists.txt

```diff
@@ -39,8 +39,8 @@ endif(NOT CMAKE_BUILD_TYPE STREQUAL "Debug")
 set(DOXYFILE_LATEX false)
 include(UseDoxygen)
 
-# Compilation
-add_definitions(-Wall)
+# Switch on warnings and disable one python interface warning.
```

Comment:
yes, I had to remove it since the gcc on jenkins does not understand W-no-defined-but-unused-templates.